osu.Game/Online/Leaderboards/PlaceholderState.cs

```diff
@@ -11,7 +11,6 @@ public enum PlaceholderState
         Unavailable,
         NoneSelected,
         NoScores,
-        NotLoggedIn,
```

Comment:
`PlaceholderState` used for `Leaderboard<>` should probably be renamed to `LeaderboardPlaceholderState` for now.